gbpservice/nfp/configurator/agents/loadbalancer_v1.py

```diff
@@ -0,0 +1,655 @@
+import os
+import oslo_messaging as messaging
+
+from gbpservice.nfp.configurator.agents import agent_base
+from oslo_log import log as logging
+from gbpservice.nfp.core.main import Event
+from gbpservice.nfp.configurator.lib import utils
+from gbpservice.nfp.configurator.lib.filter import Filter
+from gbpservice.nfp.core import poll as nfp_poll
+from gbpservice.nfp.configurator.lib import lb_constants
+
+LOG = logging.getLogger(__name__)
+
+
+class LBaasRpcSender(Filter):
+    """Agent side of the Agent to Plugin RPC API."""
+
+    API_VERSION = '2.0'
+    # history
+    #   1.0 Initial version
+    #   2.0 Generic API for agent based drivers
+    #       - get_logical_device() handling changed on plugin side;
+    #       - pool_deployed() and update_status() methods added;
+
+    def __init__(self, sc=None):
+        self.sc = sc
+        pass
+
+    def _notification(self, data):
```

Comment:
Since the notification method is common, we added new class (AgentBaseNotification) in agent_base.py file. Please create an instance of this class and invoke the _notification() method of that class.